pkg/events/events.go

```diff
@@ -5620,6 +5620,7 @@ var Definitions = eventDefinitions{
 			Sets:         []string{},
 			Params: []trace.ArgMeta{
 				{Type: "unsigned long[]", Name: "syscalls_addresses"},
+				{Type: "unsigned long", Name: "caller_context_id"},
```

Comment:
Does this id has any meaning outside the scope of tracee-ebpf?

Reply:
No but that's an internal event so the user won't get it.